go.mod

```diff
@@ -1,14 +1,13 @@
 module github.com/scroll-tech/da-codec
 
-go 1.20
+go 1.21
 
 require (
-	github.com/scroll-tech/go-ethereum v1.10.14-0.20240426041101-a860446ebaea
+	github.com/scroll-tech/go-ethereum v1.10.14-0.20240607130425-e2becce6a1a4
```

Comment:
are we able to use tags here? `scroll-v5.3.41`

Reply:
tried with v5.3.41. the tag is incompatible with golang's format. thus changing it to use commit hash.

Reply:
```
go: errors parsing go.mod:
go.mod:6:2: require github.com/scroll-tech/go-ethereum: version "v5.3.41" invalid: should be v0 or v1, not v5
```